### PR TITLE
Update matcher version and prepare to publish

### PR DESCRIPTION
**Pull request by natebosch**

### CODE REVIEW

File: pkgs/test/pubspec.yaml [hunk 1]
```diff
@@ -33,8 +33,8 @@ dependencies:
   web_socket_channel: '^1.0.0'
   yaml: '^2.0.0'
   # Use an exact version until the test_api and test_core package are stable.
-  test_api: '0.2.2'
-  test_core: '0.2.1+1'
+  test_api: '0.2.3'
```
Comment (kevmoo) [inline pkgs/test/pubspec.yaml]: Missed version bump for `pkg:test` – going to do that in a separate PR?

Comment (natebosch) [reply pkgs/test/pubspec.yaml]: Was an oversight. Fixed now and merged the changelog entry from the unpublished version.

**Comment by kevmoo** [comment]
We expect travis to blow up here, right? – because version constraints?

**Comment by natebosch** [comment]
> We expect travis to blow up here, right? – because version constraints?

I don't think so? We have overrides.
